packages/bsky/tests/views/blocks.test.ts

```diff
@@ -0,0 +1,329 @@
+    const { carol, dan } = sc.dids
+    const { data: threadAlice } = await agent.api.app.bsky.feed.getPostThread(
+      { depth: 1, uri: sc.posts[carol][0].ref.uriStr },
+      { headers: await network.serviceHeaders(dan) },
+    )
+    expect(threadAlice).toEqual({
+      thread: {
+        $type: 'app.bsky.feed.defs#blockedPost',
+        uri: sc.posts[carol][0].ref.uriStr,
+        blocked: true,
+      },
+    })
+    const { data: threadCarol } = await agent.api.app.bsky.feed.getPostThread(
+      { depth: 1, uri: sc.posts[dan][0].ref.uriStr },
+      { headers: await network.serviceHeaders(carol) },
+    )
+    expect(threadCarol).toEqual({
+      thread: {
+        $type: 'app.bsky.feed.defs#blockedPost',
+        uri: sc.posts[dan][0].ref.uriStr,
+        blocked: true,
+      },
+    })
+  })
+
+  it('blocks thread reply', async () => {
+    // Contains reply by carol
+    const { data: thread } = await agent.api.app.bsky.feed.getPostThread(
+      { depth: 1, uri: sc.posts[alice][1].ref.uriStr },
+      { headers: await network.serviceHeaders(dan) },
+    )
+    expect(forSnapshot(thread)).toMatchSnapshot()
+  })
+
+  it('blocks thread parent', async () => {
+    // Parent is a post by dan
+    const { data: thread } = await agent.api.app.bsky.feed.getPostThread(
+      { depth: 1, uri: aliceReplyToDan.ref.uriStr },
+      { headers: await network.serviceHeaders(carol) },
+    )
+    expect(forSnapshot(thread)).toMatchSnapshot()
+  })
+
+  it('blocks record embeds', async () => {
+    // Contains a deep embed of carol's post, blocked by dan
+    const { data: thread } = await agent.api.app.bsky.feed.getPostThread(
+      { depth: 0, uri: sc.posts[alice][2].ref.uriStr },
+      { headers: await network.serviceHeaders(dan) },
+    )
+    expect(forSnapshot(thread)).toMatchSnapshot()
+  })
+
+  it('errors on getting author feed', async () => {
+    const attempt1 = agent.api.app.bsky.feed.getAuthorFeed(
+      { actor: carol },
+      { headers: await network.serviceHeaders(dan) },
+    )
+    await expect(attempt1).rejects.toThrow(BlockedActorError)
+
+    const attempt2 = agent.api.app.bsky.feed.getAuthorFeed(
+      { actor: dan },
+      { headers: await network.serviceHeaders(carol) },
+    )
+    await expect(attempt2).rejects.toThrow(BlockedByActorError)
+  })
+
+  it('strips blocked users out of getTimeline', async () => {
+    const resCarol = await agent.api.app.bsky.feed.getTimeline(
+      { limit: 100 },
+      { headers: await network.serviceHeaders(carol) },
+    )
+    expect(
+      resCarol.data.feed.some((post) => post.post.author.did === dan),
+    ).toBeFalsy()
+
+    const resDan = await agent.api.app.bsky.feed.getTimeline(
+      { limit: 100 },
+      { headers: await network.serviceHeaders(dan) },
+    )
+    expect(
+      resDan.data.feed.some((post) => post.post.author.did === carol),
+    ).toBeFalsy()
+  })
+
+  // it('strips blocked users out of getPopular', async () => {
```

Comment:
Does this remain commented out intentionally?

Reply:
yea removed that method from the appview - i'll just remove this test